packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -113,47 +113,104 @@ function Cell(props) {
     intl,
   } = props;
 
+  const cellRef = useRef();
+  const [isFocusTrapEnabled, setFocusTrapEnabled] = useState(false);
   const theme = useContext(ThemeContext);
-
   const columnContext = useContext(ColumnContext);
 
-  const handleMouseDown = (event) => {
-    if (isMasked || !isSelectable) {
-      event.stopPropagation();
-      event.preventDefault();
-    } else if (onCellSelect) {
+  /**
+   * Determine if cell has focusable elements
+   */
+  const hasFocusableElements = () => {
+    const focusableElements = [...cellRef.current.querySelectorAll(
+      'a[href], button, input, textarea, select, details,[tabindex]:not([tabindex="-1"])',
+    )].filter(el => !el.hasAttribute('disabled') && !el.getAttribute('aria-hidden'));
+
+    return focusableElements.length > 0;
+  };
+
+  useEffect(() => {
+    if (isTabStop && hasFocusableElements()) {
+      columnContext.setCellAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.cell-interactable' }));
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [intl, isTabStop]);
+
+  /**
+   * Handles click event for cell
+   */
+  const onClick = useCallback(() => {
+    if (!isFocusTrapEnabled) {
```

Comment:
Should focusTrapEnabled be set to true here? I noticed while trying it out in Windows a selectable element inside the cell would reselect the cell

Reply:
@trandrew1023 - Sorry, I am not following your comment here.

Reply:
In this example https://engineering.cerner.com/terra-framework/pull/1715/components/cerner-terra-framework-docs/worklist-data-grid/examples/worklist-data-grid-focusable-cell when I was checking out the cell with dropdown, if I were to click the dropdown with mouse, the dropdown would briefly open before closing again as if the cell was selected

Reply:
This is a great find but is not caused by the logic in this JIRA.  I will log another JIRA so that this issue can be addressed.